From dd9309606fe16e95a0eb2a3319e10d4845d7756b Mon Sep 17 00:00:00 2001
From: qwang2 <quanyang.wang@windriver.com>
Date: Wed, 11 Apr 2018 13:10:25 +0800
Subject: [PATCH] gpu-viv: use DECLARE_SWAITQUEUE instead of DEFINE_SWAITER

In commit 13b35686e8b9 ("wait.[ch]: Introduce the simple
waitqueue (swait) implementation"), a new simple-wait
implementation is added and the old one has been discarded.
So use DECLARE_SWAITQUEUE intead of DEFINE_SWAITER.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c b/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c
index e03daf1..467a4e5 100644
--- a/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c
+++ b/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c
@@ -6042,7 +6042,7 @@ gceSTATUS gckOS_UserLogicalToPhysical(
             : msecs_to_jiffies(Wait);
 
 #ifdef gcdRT_KERNEL
-        DEFINE_SWAITER(wait);
+        DECLARE_SWAITQUEUE(wait);
 #else
         DECLARE_WAITQUEUE(wait, current);
         wait.flags |= WQ_FLAG_EXCLUSIVE;
-- 
1.8.5.2.233.g932f7e4

